daras_ai/text_format.py

```diff
@@ -48,3 +48,25 @@ def format_number_with_suffix(num: int) -> str:
 def unmarkdown(text: str) -> str:
     """markdown to plaintext"""
     return MarkdownIt(renderer_cls=RendererPlain).render(text)
+
+
+def wa_markdown(text: str) -> str:
+    patterns = [
```

Comment:
i think we're missing some here, like URLs `[]()` - might be better to use the markdown -> text converter with a whitelist for these https://faq.whatsapp.com/539178204879377/?cms_platform=web